src/App.js

```diff
@@ -1,25 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import HomePage from "./pages/HomePage";
+import NavBar from "./pages/NavBar";
+import { Routes, Route } from 'react-router-dom';
 
-function App() {
+const App = () => {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>My App </h1>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        {/* <Route path="/topics" element={<Topics />} />
```

Comment:
Don't try and jump ahead! You can remove any comments for unused code between commits